apps/daimo-web/src/components/buttons.tsx

```diff
@@ -1,16 +1,28 @@
 "use client";
+import { daimoLinkBase } from "@daimo/common";
+import { useState } from "react";
+
 import { detectPlatform, downloadMetadata } from "../utils/platform";
 
-export function PrimaryButton({
-  children,
-  disabled,
-}: {
-  children: React.ReactNode;
-  disabled?: boolean;
-}) {
+export function PrimaryButton({ disabled }: { disabled?: boolean }) {
```

Comment:
might want to call this something else, not just a styled button but does something specific